drivers/amazonec2/region.go

```diff
@@ -8,31 +8,34 @@ type region struct {
 	AmiId string
 }
 
-// Ubuntu 16.04 LTS 20180228.1 hvm:ebs-ssd (amd64)
+// Ubuntu 20.04 LTS hvm:ebs-ssd (amd64)
 // See https://cloud-images.ubuntu.com/locator/ec2/
 var regionDetails map[string]*region = map[string]*region{
-	"af-south-1":      {"ami-03611e0f71e9d9f66"},
-	"ap-northeast-1":  {"ami-bcb7f6da"},
-	"ap-northeast-2":  {"ami-5073de3e"},
-	"ap-southeast-1":  {"ami-41e4af3d"},
-	"ap-southeast-2":  {"ami-c1498fa3"},
-	"ap-south-1":      {"ami-1083dc7f"},
-	"ap-east-1":       {"ami-9ca0d8ed"},
-	"ca-central-1":    {"ami-8d9e19e9"},
-	"cn-north-1":      {"ami-cc4499a1"}, // Note: this is 20180126
-	"cn-northwest-1":  {"ami-fd0e1a9f"}, // Note: this is 20180126
-	"eu-central-1":    {"ami-bc4925d3"},
-	"eu-north-1":      {"ami-017ff17f"},
-	"eu-west-1":       {"ami-0b541372"},
-	"eu-west-2":       {"ami-ff46a298"},
-	"eu-west-3":       {"ami-9465d3e9"},
-	"sa-east-1":       {"ami-b5501bd9"},
-	"us-east-1":       {"ami-927185ef"},
-	"us-east-2":       {"ami-b9daeddc"},
-	"us-west-1":       {"ami-264c4646"},
-	"us-west-2":       {"ami-78a22900"},
-	"us-gov-west-1":   {"ami-2561ea44"},
-	"us-gov-east-1":   {"ami-b740a6c6"},
+	"af-south-1":      {"ami-0e5e0920562e66b27"},
+	"ap-northeast-1":  {"ami-0f08dd35fe25614c9"},
+	"ap-northeast-2":  {"ami-0688e5f8e4d3823dd"},
+	"ap-northeast-3":  {"ami-00fbd944ec4f0a3a5"},
+	"ap-southeast-1":  {"ami-0b17218a3cbd5ec4a"},
+	"ap-southeast-2":  {"ami-0eb2b6effadbcc070"},
+	"ap-south-1":      {"ami-03caa8eb4ce0612a2"},
+	"ap-east-1":       {"ami-0db28d8badd461a9b"},
+	"ca-central-1":    {"ami-0f71e3f96dbd2a805"},
+	"cn-north-1":      {"ami-0592ccadb56e65f8d"}, // Note: this is 20180126
+	"cn-northwest-1":  {"ami-007d0f254ea0f8588"}, // Note: this is 20180126
+	"eu-central-1":    {"ami-0dca0d6d4f591c2f4"},
+	"eu-north-1":      {"ami-09f4f0c83b3a83bfb"},
+	"eu-south-1":      {"ami-0d7681eb054d3e470"},
+	"eu-west-1":       {"ami-0dd0f5f97a21a8fe9"},
+	"eu-west-2":       {"ami-0925a09a52d18d09a"},
+	"eu-west-3":       {"ami-03ba9eb13b5975d61"},
+	"me-south-1":      {"ami-0b1119d3877cc4269"},
+	"sa-east-1":       {"ami-0b0cbbe7f83e5bbfd"},
+	"us-east-1":       {"ami-0fa37863afb290840"},
+	"us-east-2":       {"ami-0b5add21e87587ae1"},
+	"us-west-1":       {"ami-01d3aaa89c3c158e5"},
+	"us-west-2":       {"ami-07e573cdaa16d4e61"},
+	"us-gov-west-1":   {"ami-a7edd7c6"},
+	"us-gov-east-1":   {"ami-c39973b2"},
```

Comment:
Those two look good to me although they are not on the abovementioned website.

Reply:
Looks like they're different now so I'll go ahead and switch them out.